src/frontend/Komponenter/Behandling/VedtakOgBeregning/Skolepenger/InnvilgetSkolepenger/VedtaksformSkolepenger.tsx

```diff
@@ -130,7 +145,10 @@ export const VedtaksformSkolepenger: React.FC<{
                     behandlingId: behandling.id,
                     skoleårsperioder: skoleårsPerioderState.value,
                 },
-            }).then((res: Ressurs<IBeregningSkolepengerResponse>) => settBeregningsresultat(res));
+            }).then((res: Ressurs<IBeregningSkolepengerResponse>) => {
+                settBeregningsresultat(res);
+                settHarUtførtBeregning(true);
```

Comment:
Tror beregningsresultat her skulle kunne være at typen feilet? Burde man sjekke på om res.resultat === 'SUKSESS' ?

Reply:
Nei, for hvis det feiler så vil det komme en feilmelding isteden for beregningsresultatet, eller?

Reply:
Tror det skal komme en feilmelding da, ja?